init.lua

```diff
@@ -644,80 +586,41 @@ minetest.register_abm({
 	nodenames = {"group:molten_ore"},
 	interval = 10,
 	chance = 15,
-	action = function(pos)
-		local node = minetest.get_node_or_nil(pos)
-	
+	action = function(pos, node)
 		local cold = cools_to[node.name]
-		
-		if node == nil or cold == nil then 
-			return 
+		if cold == nil then
+			return
 		end
-	
+
 		-- don't cool near active electrodes
 		if nil ~= minetest.find_node_near(pos, 4, {mn..":electrode_on"}) then
 			return
 		end
 
-		-- don't cool near heater bricks
```

Comment:
Oh, you're right. That must have slipped out during rebasing against the last commit.
Adding it back in.